runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperConfig.java

```diff
@@ -71,21 +71,54 @@ default boolean quickhopOutOfDanger()
 	}
 
 	@ConfigItem(
-		keyName = "quickHopRegionFilter",
-		name = "Quick-hop region",
-		description = "Limit quick-hopping to worlds of a specific region",
+		keyName = "quickHopRegionFilterAU",
+		name = "Quick-hop to Australia worlds (AUS)",
+		description = "Quick-hop to worlds located in Australia",
 		position = 3
 	)
-	default RegionFilterMode quickHopRegionFilter()
+	default boolean quickHopAURegion()
 	{
-		return RegionFilterMode.NONE;
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "quickHopRegionFilterDE",
+			name = "Quick-hop to Germany worlds (DEU)",
+			description = "Quick-hop to worlds located in Germany",
+			position = 4
```

Comment:
The added configurations are indented one level too far. Please ensure all `@ConfigItem` annotations have their contents indented only one additional level.
```suggestion
		keyName = "quickHopRegionFilterDE",
		name = "Quick-hop to Germany worlds (DEU)",
		description = "Quick-hop to worlds located in Germany",
		position = 4
```